src/include/wiredtiger.in

```diff
@@ -247,6 +247,9 @@ struct __wt_cursor {
 	 * @param cursor the cursor handle
 	 * @param ... pointers to hold key fields corresponding to
 	 * WT_CURSOR::key_format.
+	 * The API will not validate the argument types passed in, caller is
```

Comment:
Better worded as: 
The API does not validate the argument types passed in, the caller is responsible for passing the correct argument types according to WT_CURSOR::key_format.

Reply:
3b8ca51